src/CaptureFile/include/CaptureFile/CaptureFileSection.h

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef CAPTURE_FILE_CAPTURE_FILE_SECTION_H_
+#define CAPTURE_FILE_CAPTURE_FILE_SECTION_H_
+
+#include <cstdint>
+
+namespace orbit_capture_file {
+
+struct CaptureFileSection {
```

Comment:
Not sure if you want to add a super brief comment here. Feel free to ignore though.